controllers/openshift_controller.go

```diff
@@ -839,6 +839,21 @@ func (r *KataConfigOpenShiftReconciler) processKataConfigDeleteRequest() (ctrl.R
 		}
 	}
 
+	scc := GetScc()
+	if scc == nil {
+		r.Log.Error(err, "error deleting SCC")
+		return ctrl.Result{Requeue: true, RequeueAfter: time.Second * 15}, nil
+	}
+	err = r.Client.Delete(context.TODO(), scc)
+	if err != nil {
```

Comment:
What if there are multiple `KataConfig` objects ? Shouldn't we do some ref counting on the SCC ?

Reply:
There won't be multiple KataConfig as it's disallowed by the webhook. At any given point there can only be a single KataConfig